OpenRA.Mods.Common/Traits/Modifiers/FrozenUnderFog.cs

```diff
@@ -67,7 +73,7 @@ bool IsVisibleInner(Actor self, Player byPlayer)
 				return self.OccupiesSpace.OccupiedCells()
 					.Any(o => byPlayer.Shroud.IsExplored(o.First));
 
-			return visible[byPlayer];
+			return stateByPlayer[byPlayer].IsVisible;
```

Comment:
This is seems to be queried, before it is initialized:

![image](https://cloud.githubusercontent.com/assets/756669/9698090/e7faa982-53a7-11e5-8b89-a098a9635841.png)